redis_schedule_job.go

```diff
@@ -98,6 +99,11 @@ func newScheduleJob(pool *ants.Pool, client redis.UniversalClient) *scheduleJob
 		seqTicker:      time.NewTicker(10 * time.Second),
 		prefix:         prefix,
 		maxLen:         maxLen,
+		scheduleErrGroupPool: &sync.Pool{New: func() any {
+			group := new(errgroup.Group)
+			group.SetLimit(2)
```

Comment:
why the SetLimit is a fixed number as `2`?

Reply:
two goroutinue were used here to handle the business, so i set 2. 
if there were used more goroutinue ,we can set 3,4,5,  like this.